safedata_validator/resources.py

```diff
@@ -1,30 +1,34 @@
 """Load and check validation resources.
 
-The `safedata_validator` package needs access to some local resources and
-configuration to work. The three main resources for file validation are:
+The `safedata_validator` package needs access to some local resources and configuration
+to work. The three main resources for file validation are:
```

Comment:
four main?

Reply:
Or simply 'main' to avoid the whole spanish inquisition thing in the future!